django/core/checks/async_checks.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from . import Error, Tags, register
+
+E001 = Error(
+    "You must not set DJANGO_ALLOW_ASYNC_UNSAFE in your "
+    "deployment environment variable",
```

Comment:
I'm adding a little more explanation to the message, and making it more consistent with the others such as the DEBUG check